### DISCUSSION TITLE
How can I receive and send high resolution and low frame rate video from server.py? I tried to change the webrtc  constraints but It appears that I need to change the paramters from server.py too.

### DISCUSSION THREAD

**Discussion by opusviz** [aiortc/aiortc#733] [Q&A]
Before filing an issue please verify the following:

* Check whether there is already an existing issue for the same topic.
* Ensure you are actually reporting an issue related to ``aiortc``. The goal
  of the issue tracker is not to provide general guidance about WebRTC or free
  debugging of your code.
* Clearly state whether the issue you are reporting can be reproduced with one
  of the examples provided with ``aiortc`` *without any changes*.
* Be considerate to the maintainers. ``aiortc`` is provided on a best-effort,
  there is no guarantee your issue will be addressed.

**Comment by opusviz** [comment]
I am testing server.py. It is working fine with webrtc default settings, image transformation and video streaming but the quality of video is very low. I am testing at webcam and sending the video back after transformation. Even without transformation, quality is not good. Frame rate is not important. I just need high resolution.

**Comment by jlaine** [comment]
aiortc will send out whatever size frames you give it.

Regarding the bitrate / frame rate there is currently no API in aiortc to control this.

**Reply by matinfazel** [reply]
Hi, Is there anything that I can do for increasing or decreasing the bitrate at the encoder?
